.cirrus.yml

```diff
@@ -0,0 +1,21 @@
+macos_instance:
+  image: ghcr.io/cirruslabs/macos-ventura-base:latest
+
+env:
+  CIRRUS_CLONE_DEPTH: '0'
+
+task:
+  name: macos-m1
+  #only_if: $CIRRUS_BRANCH == 'main' || $CIRRUS_BRANCH == 'feature/.*' || $CIRRUS_BRANCH =~ 'pull/.*'
+  environment:
+    PYTHON: '3.10'
+    TEST_SPLITS: '3'
+    TEST_GROUP: '1'
```

Comment:
Remember that this'll only run the first test group of three, so this would not test the whole test suite.

Reply:
Yeah, goal is to first get one test group running by harmonising the scripts to run for osx-64 on the GitHub Runners and osx-arm64 on Cirrus CI. 

Once that works, extending to all test types / suits / groups is the next step.

Reply:
went a different route by running only specifically marked tests via cirrus. If we find out over time that there are enough resources maybe entire test suites can be enabled.